### PR TITLE
Chatview - landscape navbar fix

### LINKED ISSUE #734: ChatView Navbar does not like landscape on "notch"-devices
image

on landscape you'll get this:

image

meh ...

### LINKED ISSUE #736: In chat view: consider removing titleview font adjustment
Discussion:

Do we want this (current state):

image

or (iOS-default):

image

Using the `adjustsFontSizeToFitWidth`-attribute on UILabels is very uncommon. In fact I can't remember that I've ever seen this on an Apple-app or any other top rated app.

I 100% prefer using the default behaviour.

### PR DESCRIPTION
**Pull request by nayooti**
closes #734 
closes #736  

Our titleView-container is actually "self-sizing", so it's width constraint is not needed.

Also removed automatic font size adjustment as discussed in #736

### CODE REVIEW

**Comment by cyBerta** [comment]
can you please check the title with location streaming enabled?

**Comment by nayooti** [comment]
> can you please check the title with location streaming enabled?

wasn't aware that. 

I fixed it by moving the location indicator out the title view. It has it's place now as a navigation item next to the badge. 

I had to make a LocationIndicator class because just assigning a plain image to the navigation item did not work (too small).

**Comment by r10s** [comment]
> But I can't find this feature on master either.

you can enable it at "Settings / Advanced / On-demand location streaming".

once done, you can send you current location to any chat using "Location" from the paper-clip-attach menu.

**Comment by nayooti** [comment]
> > But I can't find this feature on master either.
> 
> you can enable it at "Settings / Advanced / On-demand location streaming".
> 
> once done, you can send you current location to any chat using "Location" from the paper-clip-attach menu.

What I meant: after the active location streaming has ended, the location stream indicator is still visible in the navigation bar.

**Comment by r10s** [comment]
> the location stream indicator is still visible in the navigation bar.

might be some update event from the core is missing, however, at least "Attach / Stop sharing location" is working and that can be tested.

File: deltachat-ios/View/ChatTitleView.swift [hunk 1]
```diff
@@ -49,10 +38,11 @@ class ChatTitleView: UIView {
         let containerView = UIView()
         containerView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(containerView)
+
         addConstraints([ containerView.constraintAlignTopTo(self),
                          containerView.constraintAlignBottomTo(self),
                          containerView.constraintCenterXTo(self),
-                         containerView.constraintWidthTo(UIScreen.main.bounds.width - CGFloat(paddingNaviationButtons + sizeStreamingIndicator)) ])
```
Comment (cyBerta) [inline deltachat-ios/View/ChatTitleView.swift:55]: I think the removal of this line introduces the following bug:
Screen Shot 2020-05-27 at 12 27 18

Comment (nayooti) [reply deltachat-ios/View/ChatTitleView.swift:55]: Turns out this ticket produces more bugs than it actually fixes.

Comment (nayooti) [reply deltachat-ios/View/ChatTitleView.swift:55]: Thanks for finding this bug. I fixed it.

**Pull request merged**